.github/workflows/build-wheels-pypi.yml

```diff
@@ -0,0 +1,120 @@
+name: Build Cross platform / architecture wheels and upload to PyPi
+# See https://github.com/pypa/cibuildwheel for details
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "test[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Might be good to also trigger in pre-release tags? (`-alpha.N`, `-beta.N`, `aN`, `bN`)

Reply:
We are now [PEP-440](https://peps.python.org/pep-0440/#public-version-identifiers) compliant